app-admin/kio-admin/kio-admin-9999.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+KDE_ORG_CATEGORY="system"
+KFMIN=6.5.0
+QTMIN=6.7.2
+inherit ecm gear.kde.org
+
+DESCRIPTION="Manage files as administrator using the admin:// KIO protocol"
+HOMEPAGE="https://invent.kde.org/system/kio-admin"
+
+LICENSE="BSD CC0-1.0 FSFAP GPL-2 GPL-3"
+SLOT="6"
+KEYWORDS=""
+
+DEPEND="
+	>=dev-qt/qtbase-${QTMIN}:6[dbus,gui,network,xml]
```

Comment:
How did you establish `Qt6Network` and `Qt6Xml` all of a sudden?

Reply:
Included libraries that I saw kio-admin link against. With headers I only see explicit usage of dbus and widgets

```
ldd /usr/libexec/kf6/kio-admin-helper /usr/lib64/qt6/plugins/kf6/kio/admin.so /usr/lib64/qt6/plugins/kf6/kfileitemaction/kio-admin.so | grep libQt
        libQt6DBus.so.6 => /usr/lib64/libQt6DBus.so.6 (0x000070427508b000)
        libQt6Core.so.6 => /usr/lib64/libQt6Core.so.6 (0x0000704274800000)
        libQt6Network.so.6 => /usr/lib64/libQt6Network.so.6 (0x0000704273c00000)
        libQt6Xml.so.6 => /usr/lib64/libQt6Xml.so.6 (0x0000704274eb3000)
        libQt6Gui.so.6 => /usr/lib64/libQt6Gui.so.6 (0x0000704271e00000)
        libQt6DBus.so.6 => /usr/lib64/libQt6DBus.so.6 (0x0000795e76591000)
        libQt6Core.so.6 => /usr/lib64/libQt6Core.so.6 (0x0000795e75e00000)
        libQt6Network.so.6 => /usr/lib64/libQt6Network.so.6 (0x0000795e75200000)
        libQt6Xml.so.6 => /usr/lib64/libQt6Xml.so.6 (0x0000795e76498000)
        libQt6Gui.so.6 => /usr/lib64/libQt6Gui.so.6 (0x0000795e73200000)
        libQt6Gui.so.6 => /usr/lib64/libQt6Gui.so.6 (0x00007132cde00000)
        libQt6Core.so.6 => /usr/lib64/libQt6Core.so.6 (0x00007132cd600000)
        libQt6Network.so.6 => /usr/lib64/libQt6Network.so.6 (0x00007132cca00000)
        libQt6Widgets.so.6 => /usr/lib64/libQt6Widgets.so.6 (0x00007132cbc00000)
        libQt6DBus.so.6 => /usr/lib64/libQt6DBus.so.6 (0x00007132cd122000)
        libQt6Xml.so.6 => /usr/lib64/libQt6Xml.so.6 (0x00007132cd55e000)
        libQt6WaylandClient.so.6 => /usr/lib64/libQt6WaylandClient.so.6 (0x00007132ca6f3000)
```

Reply:
ldd is giving you indirect deps, only use lddtree for that.

Reply:
I see now.

So QtGui is linked due to https://github.com/KDE/kio-admin/blob/a2eb7c0783326f8f02c70fad38ea0648d5054786/src/fileaction/fileaction.cpp#L4. I see usage of QWidget though as well.

Reply:
Indeed. Do you want to fix that upstream?

Reply:
If you want to be absolutely sure, you could always build qtbase without any USE flags and gradually add USE flags to the ebuild and to the installed qtbase package, each time you build kio-admin and it complains.

I have done this a few times. It helps to save binpackages as you'll likely end up recompiling quite a bunch of times otherwise.

Reply:
> Indeed. Do you want to fix that upstream?

https://invent.kde.org/system/kio-admin/-/merge_requests/20